config/prow/config_knative.yaml

```diff
@@ -41,6 +41,13 @@ presubmits:
       args:
       - "--run-test"
       - "./test/e2e-upgrade-tests.sh"
+    - custom-test: autotls-tests
+      dot-dev: true
+      always_run: false # TODO(chaodaiG): Lets make sure it works before turn on
+      optional: true
+      args:
+      - "--run-test"
+      - "./test/e2e-auto-tls.sh"
```

Comment:
Currently this file only contains helper function. should we move the logic of executing auto tls tests into this file?
https://github.com/knative/serving/blob/master/test/e2e-auto-tls.sh

Reply:
Just finished:
https://github.com/knative/serving/pull/6968